scripts/constants.sh

```diff
@@ -25,7 +27,7 @@ current_branch=$(git symbolic-ref -q --short HEAD || git describe --tags || echo
 
 # caminogo and caminoethvm git tag and sha
```

Comment:
in line 26 this was changed from the commit 3027fac8778d592343bd7d6450fcb29cdd844db0 which is in PR #367 
it should be like this: 
![image](https://github.com/user-attachments/assets/8bdb5168-9ea1-4dd9-a703-e72301eb4b8a)